install_files/ansible-base/roles/build-securedrop-app-code-deb-pkg/files/usr.sbin.apache2

```diff
@@ -336,6 +337,7 @@
   /var/www/securedrop/translations/** r,
   /var/www/securedrop/version.py r,
   /var/www/securedrop/wordlist r,
```

Comment:
I think we don't need this line any more. There is no `/var/www/securedrop/wordlist`.